state/processor.go

```diff
@@ -187,17 +188,18 @@ func (tp *TransactionProcessor) GetLayerStateRoot(layer types.LayerID) (types.Ha
 
 // ApplyRewards applies reward reward to miners vector miners in for layer
 func (tp *TransactionProcessor) ApplyRewards(layer types.LayerID, miners []types.Address, reward *big.Int) {
+	rewardConverted := reward.Uint64()
```

Comment:
Please add a "TODO" comment here with a link to the PR where this will be fixed (#2069)